Describe Date Range Parameters (#31)
serge-andreyev/dotnet-eud: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/multi-value-and-cascading-parameters.md

---
title: Multi-Value and Cascading Parameters
author: Anna Gubareva
---
# Multi-Value and Cascading Parameters

> This document describes how to implement multi-value and cascading parameters. Cascading parameters display values that correspond to other parameters'.

Reviewer: Other parameters' what?

Author: ...other parameters' values. Added to the doc

---
title: Multi-Value and Cascading Parameters
author: Anna Gubareva
---
# Multi-Value and Cascading Parameters

> This document describes the implementation of multi-value and cascading parameters. Multi-value parameters can accept more than a single value, and cascading parameters display values corresponding to current values of other parameters.

Reviewer: ```suggestion
This document describes how to implement multi-value and cascading parameters. Cascading parameters display values that correspond to other parameters' values.
```

Author: Ok
